### PR TITLE
GUI Bugfixes

### PR DESCRIPTION
**Pull request by skaman82**
Fixes some small stuff like:
• Rework of the landing tab (fixes rendering errors of the sponsor tab, change log is better readable)
• Setup tab fix: 3D Model was overlapping when change log was open
• Fixes small css bug with tooltip icons in the config tab
• Fixes sensor icons not showing up when connecting Naze32 Acro rev6 and Sparky1.1
• Firmware flasher: scrolling down to the progress bar on click on the flash firmware button
• Small cosmetic fixes 
• Light PID Tab modification
• Placed tooltip content into jsnon file

### CODE REVIEW

**Comment by skaman82** [comment]
@hydra hope this is now done correctly :)

**Comment by skaman82** [comment]
@hydra forgot to mention this in the changelog above: Placed tooltip content into jsnon file

File: tabs/landing.css [hunk 1]
```diff
@@ -1,253 +1,252 @@
...
+#changelog .log li {
+  font-weight: normal;
+	margin-bottom:5px;
 }
 
-.text1  a:hover, .text2  a:hover, .text3 a:hover {
-    color: #59aa29;
+#changelog .log p {
+  margin-bottom: 20px;
 }
```
Comment (hydra) [inline tabs/landing.css]: spaces, not tabs please.

File: tabs/landing.html [hunk 2]
```diff
@@ -58,18 +51,16 @@
...
+   <div id="changelog">
+      <div class="button"><a id="changelog_toggle" href="#">Changelog</a></div>
+      <div class="wrapper">
+         <div class="title" i18n="defaultChangelogHead"></div>
+         <div class="log">
+            <!-- changelog content will be loaded here -->
+         </div>
+      </div>
+   </div>
 </div>
```
Comment (hydra) [inline tabs/landing.html]: spaces not tabs, indentation 4 spaces not 2.

File: tabs/led_strip.css [hunk 3]
```diff
@@ -110,15 +111,15 @@
 }
 
 .tab-led-strip .wire {
-    color: rgba(255,255,255,.5);
+	color: rgba(255,255,255,.5);
```
Comment (hydra) [inline tabs/led_strip.css]: indentation issue

File: tabs/pid_tuning.html [hunk 4]
```diff
@@ -146,6 +128,31 @@
                         </tr>
                     </table>
                 </div>
+				<div class="gui_box grey" style="margin-top:10px;">
+					<table id="pid_accel" class="pid_tuning">
+                        <tr>
+                            <th colspan="4">
+                                <div class="pid_mode">
+                                    <div style="width:25%; float:left; text-align:left;">Angle/Horizon
```
Comment (hydra) [inline tabs/pid_tuning.html]: all the english text needs to be in messages.json

File: tabs/pid_tuning.html [hunk 5]
```diff
@@ -146,6 +128,31 @@
...
+                                      <div class="helpicon cf_tip">
+                                       <div class="cf_tooltiptext" i18n="pidHelp1" style="display:none;"></div>
+                                      </div>
```
Comment (hydra) [inline tabs/pid_tuning.html]: indentation 4 spaces, no tabs

File: tabs/setup.css [hunk 6]
```diff
@@ -74,6 +74,12 @@
     background-image: url(../images/paper.jpg);
     background-size: 100%;
     background-position: center;
+	border-radius: 5px;
```
Comment (hydra) [inline tabs/setup.css]: indentation issue

**Comment by hydra** [comment]
fixed some indentation issues here, will merge this and fixes to development now.

please can you do another PR with the pid tuning messages into the messages.json file and add i18n attributes for them?

**Comment by hydra** [comment]
I also noticed two other issues in testing:

1) buttons on dataflash dialogs do not render correct.  They are usable but the borders have gone.

issue-1a
issue-1b

2) when using RX_MSP the controls and buttons look mismatched in style compared to the main window.

issue-2

**Comment by hydra** [comment]
I noticed another minor issue too.

3) When making the window small the some buttons on the left are not visible.

issue-3

Suggest making the button area scrollable when it becomes too small, perhaps using `overflow: scroll` or something.

**Comment by hydra** [comment]
Issue 3 above is only an issue when the log is visible.

**Comment by skaman82** [comment]
@hydra Thanks for noticing. Will test this and do some fixing in the next PR 

Von meinem iPhone gesendet

**Comment by skaman82** [comment]
@hydra Fixed all mentioned things above with PR #277

**Pull request merged**